### PR TITLE
docs: add react-native-web support plugin

### PR DESCRIPTION
**Pull request by yamankatby**
### Summary
- Created a Docusaurus plugin to run `react-native` components (using React Native for the Web)
- Added a simple components overview example

https://user-images.githubusercontent.com/35961879/215330302-f8cbf7b2-7fba-4718-be86-71b6e521ae19.mov

### CODE REVIEW

**Review by Drakeoon — CHANGES_REQUESTED** [review]
Overall the changes look good, however please take a look at one comment I've left

File: docusaurus/package.json [hunk 1]
```diff
@@ -27,7 +27,10 @@
     "prism-react-renderer": "^1.3.5",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "react-native-vector-icons": "^9.2.0"
+    "react-native-vector-icons": "^9.2.0",
+    "react-native-paper": "^5.1.4",
```
[the remark this answers is not in the document]
Comment (yamankatby) [reply docusaurus/package.json:31]: hmmm, taking a look 👀

[the remark this answers is not in the document]
Comment (yamankatby) [reply docusaurus/package.json]: well, we are good now 🔥

**Comment by lukewalczak** [comment]
@yamankatby looks like the build is not passing currently, please take a look 🙏🏽

**Comment by yamankatby** [comment]
@lukewalczak @Drakeoon, all done take a look 👀

**Comment by lukewalczak** [comment]
Awesome, changes looks good! I will try to play with it in the evening and then merge ✌🏽

**Comment by lukewalczak** [comment]
I've just run docs locally and while on google chrome I don't see any problems, icons don't load on the safari:

image

**Comment by yamankatby** [comment]
I'll take a look 👀

**Comment by yamankatby** [comment]
Hey @lukewalczak,
Well, looks ok here. I'm not sure what could it be, maybe a hard refresh can make it, if not can you please see if any logs in the console?

Screenshot 2023-01-31 at 11 40 38

**Comment by lukewalczak** [comment]
Tested again on mobile and web Safari browsers and the result is satisfying. Thanks for adjusting all the requests!

**Review by lukewalczak — APPROVED** [review]

**Pull request merged**